docs/content/Configuration/Connecting-to-the-Database.mdx

```diff
@@ -178,10 +184,28 @@ highlighted as such in their respective pages.
 
 You're more than welcome to contribute new drivers as well as new features and
 patches to
-[existing drivers](https://github.com/cube-js/cube/tree/master/packages).
-Please check the
+[existing drivers](https://github.com/cube-js/cube/tree/master/packages). Please
+check the
 [contribution guidelines](https://github.com/cube-js/cube/blob/master/CONTRIBUTING.md#contributing-database-drivers)
 and join the `#contributors` channel in our
 [Slack community](https://slack.cube.dev).
 
+## Cube Cloud
+
+### <--{"id" : "Cube Cloud"}--> Allowing connections from Cube Cloud IP
+
+In some cases you'd need to allow connections from your Cube Cloud deployment IP
+address to your database. You can copy the IP address from either the Database
+Setup step in deployment creation, or from **Settings / Configuration** in your
+deployment.
+
+### <--{"id" : "Cube Cloud"}--> Connecting to data sources not exposed over the Internet
+
+Cube Cloud supports connecting to data sources within private VPCs. Please
+[contact us][cube-contact] to learn more. If you already have VPCs
+enabled in your account, check out the [VPC documentation][ref-cloud-conf-vpc]
+to learn how to get started.
+
+[cube-contact]: https://cube.dev/contact/
```

Comment:
@hassankhan Given the length of the page, this is undiscoverable. Let's remove it from here and copy to every individual database page instead; some of them—e.g., Redshift—already have something along these lines.